src/dstr-assignment/obj-rest-computed-property.case

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: >
+    Destructuring field can be a computed property, i.e it can be defined 
+    only at runtime. Rest operantion needs to skip these properties as well.
+template: default
+esid: pending
+includes: [propertyHelper.js]
+features: [object-rest]
+---*/
+
+//- setup
+var a = "foo";
+//- elems
+{[a]:b, ...rest}
+//- vals
+{ foo: 1, bar: 2, baz: 3 }
+//- body
+assert.sameValue(b, 1);
+assert.sameValue(rest.bar, 2);
+assert.sameValue(rest.baz, 3);
+
+verifyEnumerable(rest, "bar");
+verifyWritable(rest, "bar");
+verifyConfigurable(rest, "bar");
```

Comment:
you can use the new `verifyProperty`:

```js
verifyProperty(rest, "bar", {
  enumerable: true,
  writable: true,
  configurable: true
});
```

It would also be nice to verify `rest` has not property `foo`.

Out of curiosity: do we have similar cases for dstr-binding too? A `let {[a]:b, ...rest} = vals` would be nice. I need to check this.